src/index.html

```diff
@@ -11,6 +11,44 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form action="https://mate-academy-form-lesson. herokuapp.com/create-application"
+      method="post"
+      class="search_bar"
+      data-qa="big"
+    >
```

Comment:
```suggestion
    <form
      action="https://mate-academy-form-lesson. herokuapp.com/create-application"
      method="post"
      class="search_bar"
      data-qa="big"
    >
```

Reply:
thank you for the correction
I will keep it in mind